token/cli/src/main.rs

```diff
@@ -3768,8 +3785,10 @@ async fn process_command<'a>(
             let decimals = value_t_or_exit!(arg_matches, "decimals", u8);
             let mint_authority =
                 config.pubkey_or_default(arg_matches, "mint_authority", &mut wallet_manager)?;
-            let memo = value_t!(arg_matches, "memo", String).ok();
+            let memo: Option<String> = value_t!(arg_matches, "memo", String).ok();
```

Comment:
nit: Can you revert this change?

Reply:
a push with the changes removed should do the trick, I think